src/zenml/cli/integration.py

```diff
@@ -359,11 +368,27 @@ def uninstall(
     for integration_name in integrations:
         try:
             if integration_registry.is_installed(integration_name):
-                requirements += (
-                    integration_registry.select_integration_requirements(
-                        integration_name
+                # TODO: keep only `else` branch once 3.8 is deprecated
+                if (
```

Comment:
This is a quite interesting catch. As far as I understand, we have to keep the `typing-extensions` even if we uninstall the `tensorflow` integration over the CLI otherwise it will lead to issues.

I can think of another example that might need to adjust similarly to the `uninstall` command. `mlflow` now has `pydantic` V2 as a requirement, otherwise when installed through the CLI, it downgrades it to V1. So, in its current state, if I uninstalled the `mlflow` integration through the CLI, it will remove pydantic altogether 🥲 Would be amazing to fix that here as well.

Reply:
I generalized the concept and added more ignores by going through the integration list. I will re-request reviews, since it is a bold move 🙂